src/test/java/org/apache/maven/plugins/dependency/tree/TestTreeMojo.java

```diff
@@ -137,6 +147,74 @@ public void testTreeTGFSerializing() throws Exception {
         assertTrue(findString(contents, "testGroupId:release:jar:1.0:compile"));
     }
 
+    /**
+     * Test the JSON format serialization on DependencyNodes with circular dependence
+     */
+    public void testTreeJsonCircularDependency() throws Exception {
```

Comment:
can you declare a more specific exception?

Reply:
This has been updated to `IOException`.